### PR TITLE
90 lesson link component

### PR DESCRIPTION
**Pull request by Parham2002**
Component created following figma styling. 
Heading and duration passed down as props.
SVGs added as images from assets.
Link added to heading and arrow image.

### CODE REVIEW

File: src/components/LessonLink/LessonLink.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  &__arrow {
+    height: 3.125rem;
+    width: 3.125rem;
+    border-radius: 50%;
+    background-color: rgba($color-blue-navy-dark, 0.1);
+    color: rgba($color-blue-navy-dark, 0.1);
+    display: flex;
+    margin-top: 0.063px;
```
Comment (BrookeVerse) [inline src/components/LessonLink/LessonLink.scss:47]: Can you please made sure any pxs are converted to REM please

File: src/components/LessonLink/LessonLink.test.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    const { container } = customRender(<LessonLink />);
+    expect(container).toMatchSnapshot();
+  });
+
+  it("matches the snapshot taken with props", () => {
+    const { container } = customRender(
+      <LessonLink courseDuration={"10 min"} courseTitle={"blablabla"} />
+    );
+    expect(container).toMatchSnapshot();
+  });
```
Comment (BrookeVerse) [inline src/components/LessonLink/LessonLink.test.jsx:17]: Can you change this to just one snapshot using the props, dont need both :D

**Review by BrookeVerse — CHANGES_REQUESTED** [review]
Please make changed from comments please :D

File: src/components/LessonLink/LessonLink.scss (hunk 1 above)
Comment (Parham2002) [reply src/components/LessonLink/LessonLink.scss]: converted px to rem

File: src/components/LessonLink/LessonLink.test.jsx (hunk 2 above)
Comment (Parham2002) [reply src/components/LessonLink/LessonLink.test.jsx:17]: deleted one prop.

Comment (Parham2002) [reply src/components/LessonLink/LessonLink.test.jsx]: 50% of all snapshots snapped out of existence

**Review by BrookeVerse — APPROVED** [review]

**Review by karinhawk — APPROVED** [review]

**Pull request merged**